src/quad_gl.rs

```diff
@@ -783,12 +826,10 @@ impl QuadGl {
             }
             ctx.apply_bindings(&bindings);
 
-            ctx.apply_uniforms(&shader::Uniforms {
-                projection: dc.projection,
-                model: dc.model,
-                time,
-                data: pipeline.uniforms_data.clone(),
-            });
+            pipeline.set_uniform("Projection", dc.projection);
+            pipeline.set_uniform("Model", dc.model);
+            pipeline.set_uniform("_Time", time);
+            ctx.apply_uniforms_buffer(&pipeline.uniforms_data[..]);
```

Comment:
Also, to use this line miniquad should be updated. I should add new version to `Cargo.toml`?